### DISCUSSION TITLE
Control over errors/messages that are printed to the browser console from the wasm runtime

### DISCUSSION THREAD

**Discussion by dlemstra** [dotnet/runtime#110046] [Q&A] [answered]
We are using the wasm runtime with `const runtime = await dotnet.create()` and are using this to run dotnet code in our Angular application. Sometimes the wasm runtime prints messages or errors to the browser console. One case where this happens is when our application runs out of memory. Are there plans or is it already possible to insert a hook in the method that prints this to the console?

**Comment by maraf** [comment, marked as answer]
You can override functions that print to console with https://github.com/dotnet/runtime/blob/main/src/mono/wasm/testassets/WasmBasicTestApp/App/wwwroot/main.js#L84-L93

**Reply by dlemstra** [reply]
Thanks for your response and a link to an example!
